<launch>
  <node pkg="rosbag" type="play" name="rosbag" args="
  $(find styx)../../../data/dbw_test.rosbag.bag /vehicle/throttle_cmd:=/actual/throttle_cmd /vehicle/steering_cmd:=/actual/steering_cmd /vehicle/brake_cmd:=/actual/brake_cmd" required="true"/>
  <node pkg="twist_controller" type="dbw_test.py" name="dbw_test"/>
  <include file="$(find twist_controller)/launch/dbw.launch"/>
</launch>
